src/main.js

```diff
@@ -0,0 +1,18 @@
+import IndexPageModes from './layouts/IndexPageModes';
+import { addListenerToButtons } from './changeGameMode';
+
+const indexPage = new IndexPageModes();
+
+document.getElementById('main-grid-container').appendChild(indexPage.render());
+
+document.addListenerToButtons = addListenerToButtons(
+  ['.btn-characters', '.btn-vehicles', '.btn-starships'],
+  '.main__header',
+  '.main__description',
```

Comment:
Could `main__header` and `main__description` be used directly in `changeGameMode` like this:

```
const changeGameMode = (text, description) => {
  document.querySelector('.main__header').innerHTML = `MODE: ${text}`;
  document.querySelector(
    '.main__description'
  ).innerHTML = `You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select ${description}`;
};
```
It seems the classes of these elements should remain constant so I wonder if it's needed to leave a possibility to pass some other arguments to `addListenerToButtons`.